docusaurus/docs-internals/editor-controls/data-controls/multiselect.md

```diff
@@ -0,0 +1,278 @@
+  ],
+}
+```
+
+#### Devices examples
+It will be rendered on all devices. This value can be skipped because it is set to `"all"` by default.
+
+```js
+{
+  id: "pageTabs",
+  type: "multiSelect",
+  devices: "all",
+  choices: [
+    { title: "Timeline", value: "timeline" },
+    { title: "Events", value: "events" },
+  ]
+}
+```
+
+Rendering will occur only on `desktop`.
+
+```js
+{
+  id: "pageTabs",
+  type: "multiSelect",
+  devices: "desktop",
+  choices: [
+    { title: "Timeline", value: "timeline" },
+    { title: "Events", value: "events" },
+  ]
+}
+```
+The display is limited to responsive modes, specifically `tablet` and `mobile`.
+
+```js
+{
+  id: "pageTabs",
+  type: "multiSelect",
+  devices: "responsive",
+  choices: [
+    { title: "Timeline", value: "timeline" },
+    { title: "Events", value: "events" },
+  ]
+}
+```
+
+#### Disabled examples
+
+Control will be disabled. Normally, here should be your dynamic condition.
+
+```js
+{
+  id: "pageTabs",
+  type: "multiSelect",
+  disabled: true,
+  choices: [
+    { title: "Timeline", value: "timeline" },
+    { title: "Events", value: "events" },
+  ]
+}
+```
+
+Control will be disabled when `videoType` variable will be `"custom"`.
+`getValue` is a getter function that allows us to retrieve the value of controls by their id.
+`"videoType"` is the id of the `"select"` control below.
+
+```js
+const getToolbarContols = ({ getValue }) => {
+  const videoType = getValue("videoType");
+
+  return [
+    {
+      id: "videoType",
+      type: "select",
+      choices: [
+        { title: "Youtube", value: "youtube" },
+        { title: "Custom", value: "custom" }
+      ]
+    },
+    {
+      id: "pageTabs",
+      type: "multiSelect",
+      disabled: videoType === "custom",
+      choices: [
+        { title: "Timeline", value: "timeline" },
+        { title: "Events", value: "events" },
+      ]
+    }
+  ]
+}
+```
+
+#### Helper examples
+
+The helper object contains a content property with the value `"Helper"`, which will be displayed as additional guidance or information for the user.
+
+```js
+{
+  id: "pageTabs",
+  type: "multiSelect",
+  choices: [
+    { title: "Timeline", value: "timeline" },
+    { title: "Events", value: "events" },
+  ],
+  helper: {
+    content: "Helper"
+  }
+}
+```
+
+When the helper object contains a position property with the value `"top-start"`, it indicates that the helper text will be displayed at the top left corner of the icon.
+
+```js
+{
+  id: "pageTabs",
+  type: "multiSelect",
+  choices: [
+    { title: "Timeline", value: "timeline" },
+    { title: "Events", value: "events" },
+  ],
+  helper: {
+    content: "Helper",
+    position: "top-start"
+  }
+}
+```
+
+#### Async choices example
+
+`loadChoices`: This function is an asynchronous function designed to fetch a list of choices from a given URL by `Value`. The function takes an AbortSignal as an argument, which allows the fetch operation to be aborted if necessary.
```

Comment:
`loadChoices`: This function is an asynchronous function created to fetch a list of choices from a given URL by `value`.
`/*  here need to explain value from where we get it  */`
The function takes an `AbortSignal` as an argument, which allows the fetch operation to be aborted if necessary.